itez/beneficiary/models.py

```diff
@@ -1,4 +1,5 @@
 from django.contrib.gis.db import models
+from django.db.models.fields import CharField
```

Comment:
we dont need to import this as we get it from `models.CharField()`

Reply:
Not sure where that came from, it could be from code suggestion plugins